packages/flutter_tools/lib/src/version.dart

```diff
@@ -405,6 +414,92 @@ class FlutterVersion {
   }
 }
 
+/// Checks if the provided [version] is tracking a standard remote.
+///
+/// A "standard remote" is one having the same url as [flutterGitUrl] or
```

Comment:
TODO(RoyARG02): Explicitly specify that we support **only** the HTTPS & SSH urls as provided by GitHub to be standard remotes.